source/api/cypress-api/config.md

```diff
@@ -117,12 +120,13 @@ Cypress.config() // => {defaultCommandTimeout: 10000, viewportHeight: 900, ...}
 
 # Notes
 
-**Why use `Cypress.config` instead of `cy.config`?**
+**Why is it `Cypress.config` and not `cy.config`?**
 
 As a rule of thumb anything you call from `Cypress` affects global state. Anything you call from `cy` affects local state.
 
-Methods on `cy` are local and specific to a single test. Side effects from `cy` methods are restored between each test. We chose to use `Cypress` because changes to your configuration options take effect for the remainder of **ALL** tests.
+Since the configuration added or changed by `Cypress.config` is only in scope for the current spec file, you'd think that it should be `cy.config` and not `Cypress.config`&hellip;and you'd be right. The fact that `Cypress.config` affects local state is an artifact of the API evolving over time: `Cypress.config` used to affect global state&mdash;configuration added in one test spec file was available in other specs&mdash;but the Cypress team wisely made each spec run in isolation in {% url `3.0.0` changelog#3-0-0 %} and by that time `Cypress.config` was public API.
```

Comment:
Not totes sure about adding such a comment, but it's legit and it's squirrelled away in the _Notes_ section at the bottom so it's not _in your face when you're reading the documentation as reference.

> Any thoughts to adding a `cy.config` command and deprecating `Cypress.config`? (Same for `cy.env` and `Cypress.env`?)